x-pack/plugins/apm/public/components/app/TraceOverview/TraceList.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Transaction } from '../../../../typings/Transaction';
+import { ITransactionGroup } from '../../../../typings/TransactionGroup';
+import { fontSizes, truncate } from '../../../style/variables';
+// @ts-ignore
+import { asMillisWithDefault } from '../../../utils/formatters';
+import { ImpactBar } from '../../shared/ImpactBar';
+import { ITableColumn, ManagedTable } from '../../shared/ManagedTable';
+// @ts-ignore
+import TooltipOverlay from '../../shared/TooltipOverlay';
+import { TraceLink } from '../../shared/TraceLink';
+
+function formatString(value: string) {
+  return value || 'N/A';
+}
```

Comment:
This is unnecessary.  With typescript we know that `transaction.transaction.name` and `transaction.context.service.name` will always be defined.

Reply:
Sounds good -- I can clean these little things up in my beta ticket so we don't make this PR restart CI too many more times before getting it into master. :)